pkg/jx/cmd/cmd_test_helpers/app_test_helpers.go

```diff
@@ -275,3 +279,120 @@ func CreateAppTestOptions(gitOps bool, t *testing.T) *AppTestOptions {
 	return &o
 
 }
+
+// CreateAppTestOptionsForApp configures the mock environment for running apps related tests for a given App
```

Comment:
This seems to repliacate a lot of `CreateAppTestOptions` - can we just pass this in as an option to the other function, and ignore if not set?